src/styles/index.scss

```diff
@@ -1,3 +1,70 @@
+$calendar-width: 726px;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 30px;
+  font-family: Arial, sans-serif;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $calendar-width;
+  padding: 10px;
+}
+
+$day-of-week: (
```

Comment:
Interesting approach with indexes. :) 
Why have you used three digits for index, instead of just one? 
I mean, looks like just 0 - 6 would have worked just fine.